### DISCUSSION TITLE
Support of aocoda f405 v1.1

### DISCUSSION THREAD

**Discussion by wittzer** [iNavFlight/inav#8588] [Q&A]
A newbie question, I got the aocoda f405 v1.1 (officially supported by betaflight) and I wish to load INAV. 
Can I simply load the generic INAV firmware and do pin mapping via CLI or where can I learn to edit inav code and compile custom firmware that support my FC?

Here are the FC https://www.aliexpress.com/i/1005004707206134.html , it say support inav but i cant find any inav firmware that compatible and seller never answer my question.

**Comment by stronnag** [comment]
There is no resource mapping in INAV.

There is a basic [wiki article](https://github.com/iNavFlight/inav/wiki/Building-custom-firmware) on building custom firmware. The `AOCODARCF4V2` target might be a good starting place.

**Reply by IsmailOpu1** [reply]
does it supports the INav firmware now?

**Comment by alex-441** [comment]
Hi! Do anyone still need it? I could try to make custom build, but I have no such board to test and verify.
